File: app/src/main/java/com/kota/Bahamut/Command/BahamutCommandLoadArticleEndForSearch.java
package com.kota.Bahamut.Command;

import com.kota.Telnet.Reference.TelnetKeyboard;
import com.kota.Telnet.TelnetClient;

public class BahamutCommandLoadArticleEndForSearch {
    public void execute() {
        System.out.print(toString() + " ");
        TelnetClient.getClient().sendKeyboardInputToServer(TelnetKeyboard.END);
    }

    public String toString() {
        return "[LoadArticleEndForSearch]";
    }
}

File: app/src/main/java/com/kota/Bahamut/command/BahamutCommandLoadArticleEndForSearch.kt
package com.kota.Bahamut.command

import com.kota.telnet.reference.TelnetKeyboard
import com.kota.telnet.TelnetClient

class BahamutCommandLoadArticleEndForSearch {
    fun execute() {
        print(toString() + " ")
        TelnetClient.client?.sendKeyboardInputToServer(TelnetKeyboard.END)
    }

    override fun toString(): String {
        return "[LoadArticleEndForSearch]"
    }
}
